spyder/plugins/ipythonconsole/plugin.py

```diff
@@ -76,6 +77,10 @@
 dependencies.add("matplotlib", _("Display 2D graphics in the IPython Console"),
                  required_version=MATPLOTLIB_REQVER, optional=True)
 
+# CSS style
+PLUGINS_PATH = get_module_source_path('spyder', 'plugins')
+CSS_PATH = osp.join(PLUGINS_PATH, 'help', 'utils', 'static', 'css')

```

Comment:
There's no need for these two lines because (at the end) CSS_PATH is going to be passed from the main window to this plugin, so please remove them.